platform/pulp/tasking/tasks.py

```diff
@@ -96,7 +96,7 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
     try:
         celery.tasks[name].apply_async(*inner_args, **inner_kwargs)
```

Comment:
I believe this is correct as is. inner_kwargs in this case needs to be unpacked, since it's:
```
async_kwargs = {
            "args": args,
            "kwargs": kwargs,
}
```